Assets/MRTK/Providers/XRSDK/Controllers/GenericXRSDKController.cs

```diff
@@ -145,16 +145,36 @@ protected virtual void UpdateButtonData(MixedRealityInteractionMapping interacti
                 {
                     interactionMapping.BoolData = !Mathf.Approximately(triggerData, 0.0f);
                 }
+                else if (interactionMapping.InputType == DeviceInputType.GripTouch
+                    && inputDevice.TryGetFeatureValue(CommonUsages.grip, out float gripData))
+                {
+                    interactionMapping.BoolData = !Mathf.Approximately(gripData, 0.0f);
+                }
                 else
                 {
                     InputFeatureUsage<bool> buttonUsage;
 
                     // Update the interaction data source
+                    // We need to update the DeviceInputType to differentiate between primary/secondary/other buttons
+                    // For now it maps directly to the primary button as a stopgap measure
```

Comment:
It would be helpful to link to an issue or something that explains more about what the stopgap is/plans for how to unstopgap. Otherwise this will become a scary comment that loses context pretty quickly going forward.

Reply:
I should say, I'm super happy that you left a comment here because it highlights the why really well, I think it just needs a little more info.

Reply:
Part of the enum refactor work hopefully fixed this issue!